frontend/src/components/requestChanges/SummaryDeclarationChangeRequest.vue

```diff
@@ -488,9 +497,7 @@ export default {
       'setDeclarationModel',
     ]),
     expandAllPanels() {
-      for (let i = 0; i < this.numberOfPanelsToExpand; i++) {
```

Comment:
I believe the numberOfPanelsToExpand() function can be removed in this file as well.

Reply:
Good call: https://github.com/bcgov/EDUC-CCOF/pull/561/commits/1934d93a12131217722f85a852dbe850e420cd52